README.md

```diff
@@ -3,9 +3,10 @@
 A set of scripts to automatically deploy a [BinderHub](https://binderhub.readthedocs.io/en/latest/index.html) onto [Microsoft Azure](https://azure.microsoft.com/en-gb/) and connect a [DockerHub](https://hub.docker.com/) container registry.
 
 **List of scripts:**
-* [**logs.sh**](#logs.sh)
-* [**info.sh**](#info.sh)
-* [**teardown.sh**](#teardown.sh)
+* [**deploy.sh**](#deploysh)
+* [**logs.sh**](#logssh)
+* [**info.sh**](#infosh)
+* [**teardown.sh**](#teardownsh)
 
 ## Usage
 
```

Comment:
I know this might seem trivial but it might be worth adding how to run each of the scripts (helpful for newbies of people having brain farts like me)

And perhaps adding a "this is the order you'd run everything"

Reply:
I think I have the `chmod` command documented on the branch for `setup.sh` but I can add it here. Worst that can happen is a merge conflict!

Reply:
Done in [`52f38dd`](https://github.com/alan-turing-institute/binderhub-deploy/pull/5/commits/52f38dda1b9d3f2238ef6e4ad33d53ef34993089)